Venmo/src/main/java/com/braintreepayments/api/VenmoClient.java

```diff
@@ -428,6 +447,219 @@ private Intent getLaunchIntent(Configuration configuration, String profileId, St
         return venmoIntent;
     }
 
+    /**
+     * Use this method with the manual browser switch integration pattern.
+     *
+     * @param browserSwitchResult a {@link BrowserSwitchResult} with a {@link BrowserSwitchStatus}
+     * @param callback            {@link VenmoOnActivityResultCallback}
+     */
+    public void onBrowserSwitchResult(@NonNull BrowserSwitchResult browserSwitchResult, @NonNull final VenmoOnActivityResultCallback callback) {
```

Comment:
These new methods are only required for the app links flow (in the case of a web fallback) right? If so, can we call that out in the docstrings that if you opt-in to app links, you must implement these methods?

Reply:
These methods would be for both the web fallback and app switch flows using app links, but _only_ for merchants using the manual browser switch pattern. The app links flow with app installed also uses this method (we get a different result back from the app links flow for both app install and web fallback).

Reply:
Gotcha! So maybe in our `VenmoRequest.useWebFallback` docstring we can call out that these must be implemented

Reply:
Yeah, we can call that out there for merchants using the manual browser switch pattern. Updated: https://github.com/braintree/braintree_android/pull/911/commits/33063cd16c6cfd7ac732913a0be99e7e6df7f51f